ray-operator/config/default/kustomization.yaml

```diff
@@ -1,5 +1,5 @@
 # Adds namespace to all resources.
-namespace: ray-system
```

Comment:
Too late to change this now, but I always thought it was better practice to use `ray-system` namespace for kuberay instead of `default`. We probably should have changed the default namespace for helm to use `ray-system`, instead of changing this to default namespace.

Reply:
This is also a breaking change for anyone previously using kustomize to install kuberay, wondering if we should consider reverting

Reply:
@kevin85421 @MortalHappiness what do you think?

Reply:
Sorry for the inconvenience. However, this change is from @kevin85421. it might wait for discussion.

Reply:
Hi @andrewsykim, I recall that we discussed this in early 2024. My point is that I have met several users who are new to K8s, and they don't know they need to specify a namespace with `kubectl get pods -n ray-system`. Placing it in the default namespace can help avoid these kinds of user questions. For me, the top priority for documentation and examples is to ensure that most users can follow along.

If there are no clear benefits (e.g., stability) to setting `ray-system` as the default, I prefer using the `default` namespace.

Reply:
> If there are no clear benefits (e.g., stability) to setting ray-system as the default, I prefer using the default namespace.

I think the clear benefit is that in practice (and in production), you shouldn't use the default namespace. By setting it as the default, we are making it harder for users to start with best practices. 

> My point is that I have met several users who are new to K8s, and they don't know they need to specify a namespace with kubectl get pods -n ray-system. Placing it in the default namespace can help avoid these kinds of user questions

I understand this, but I feel like this is a documentation problem.

Reply:
> I think the clear benefit is that in practice (and in production)

"Production" is not the main goal of the documentation. The primary goal is to be easy to follow. Maybe we can have a separate guide for "production best practices." instead?

> but I feel like this is a documentation problem.

We can discuss this tomorrow. In my experience, "namespace" is an unknown unknown for these users (perhaps data scientists or MLEs), and it's not easy for them to find the corresponding documentation.

Let's discuss this tomorrow.

Reply:
Sounds good -- I think a production guide that covers this and other areas is probably warranted anyways

Reply:
+1 for production best practice documentation.